fix: fix beberapa feedback user

kcraksa opened the pull request.

kcraksa committed:
fix: fix beberapa feedback user

adisatr7 commented on .env.example:
```diff
@@ -1,6 +1,6 @@
-VITE_SUPABASE_PROJECT_ID=
-VITE_SUPABASE_URL=
+VITE_SUPABASE_PROJECT_ID=zupygwgwsrcwhkwhuwtk
```
Value `.env` **JANGAN** pernah di-commit 😅 Ini ilmu dari mana?

adisatr7 commented on README.md:
````diff
@@ -1,383 +1,183 @@
-   updated_at timestamp with time zone DEFAULT now(),
-   nidn text,
-```
+# Supabase CLI
````
@kcraksa bisa jelasin tujuan perubahan di readme.md ini biar apa?

adisatr7 commented on supabase/functions/create-user/index.ts:
```diff
@@ -0,0 +1,172 @@
+import { createClient } from "https://esm.sh/@supabase/supabase-js@2.38.4";
+
+const corsHeaders = {
```
Reuse CORS header yg sudah ada di folder `_shared`. Kalo memang kurang cukup, edit CORS header di file itu saja.

adisatr7 commented on supabase/functions/delete-user/index.ts:
```diff
@@ -0,0 +1,133 @@
+import { createClient } from "https://esm.sh/@supabase/supabase-js@2.38.4";
+
+const corsHeaders = {
```
Reuse CORS header di  folder `_shared`. Prinsip DRY-nya dipake (Don't Repeat Yourself)

adisatr7 commented on supabase/functions/update-user/index.ts:
```diff
@@ -0,0 +1,149 @@
+import { createClient } from "https://esm.sh/@supabase/supabase-js@2.38.4";
+
+const corsHeaders = {
```
Reuse CORS header di folder `_shared`

adisatr7 commented on vite.config.ts:
```diff
@@ -8,6 +8,13 @@ export default defineConfig(({ mode }) => ({
     port: 8080,
+    proxy: {
+      '/functions': {
+        target: 'https://zupygwgwsrcwhkwhuwtk.supabase.co',
```
Gk boleh gini. Supabase tiap environment link-nya beda dan blm tentu pake ini.

adisatr7 reviewed (changes requested):
Revisi dulu ya.

Tambahan selain yg udah dijelaskan di bawah: Untuk setiap penambahan Supabase function baru, tolong tambahkan function-nya ke `.github/workflows/supabase-functions.yml` agar **auto deploy** ke Supabase client

kcraksa committed:
fix: improve login flow to use email-based authentication

- Change login to query users by email instead of ID
- Add sync_user_id_by_email function to handle ID synchronization
- Update RLS policies to allow email-based user lookup
- Make sync function idempotent to handle multiple calls
- Fix text in signing dialog (kunci digital instead of sertifikat)
- Add migrations for user ID sync functionality

kcraksa committed:
fix

kcraksa committed:
fix feedback 2

kcraksa committed:
fixing feedback

kcraksa committed:
fix feedback

kcraksa committed:
make ijazah dynamic

adisatr7 commented:
Author refuses to address the issues mentioned in the PR reviews and instead introduces more AI slop changes, committing random new files, making unnecessary edits, and even committing environment secrets into git. And for that reason, this PR is rejected.
